src/main/java/io/cdap/plugin/sfmc/source/MarketingCloudInputFormat.java

```diff
@@ -119,7 +126,7 @@ static List<MarketingCloudObjectInfo> fetchTableInfo(SourceQueryMode mode,
   /**
    * Fetch the fields for passed object.
    */
-  private static MarketingCloudObjectInfo getTableMetaData(SourceObject object, String dataExtensionKey,
+  public static MarketingCloudObjectInfo getTableMetaData(SourceObject object, String dataExtensionKey,
```

Comment:
why public?

Reply:
Because i am calling the getMetaData method in MarketingConnectorConfig class in getSchema Method. Without making the access modifier public, I will not be able to access this method